contracts/src/bin/vendor.rs

```diff
@@ -33,19 +33,21 @@ fn run() -> Result<()> {
         )?
         .github(
             "BalancerV2Vault",
-            "balancer-labs/balancer-v2-monorepo/a3b570a2aa655d4c4941a67e3db6a06fbd72ef09/\
-             pkg/deployments/deployed/mainnet/Vault.json",
+            "balancer-labs/balancer-subgraph-v2/master/abis/Vault.json",
         )?
         .github(
             "BalancerV2WeightedPoolFactory",
-            "balancer-labs/balancer-v2-monorepo/a3b570a2aa655d4c4941a67e3db6a06fbd72ef09/\
-             pkg/deployments/deployed/mainnet/WeightedPoolFactory.json",
+            "balancer-labs/balancer-subgraph-v2/master/abis/WeightedPoolFactory.json",
         )?
         .github(
             "BalancerV2WeightedPool2TokensFactory",
             "balancer-labs/balancer-v2-monorepo/a3b570a2aa655d4c4941a67e3db6a06fbd72ef09/\
              pkg/deployments/deployed/mainnet/WeightedPool2TokensFactory.json",
         )?
+        .github(
+            "BalancerV2StablePoolFactory",
+            "balancer-labs/balancer-subgraph-v2/master/abis/StablePoolFactory.json",
```

Comment:
Can we use a tag or commit hash? While they have re-written commit history, I still think using a specific commit is important and more reproducible in general.

Reply:
I thought that I had pointed out somewhere that this repo does not have any tags or releases so that this wouldn't be possible. Perhaps the link to this discussion got lost when creating the new PR I will find it.

Reply:
If the repo doesn't have tags, you can always use commit hashes:
https://raw.githubusercontent.com/balancer-labs/balancer-subgraph-v2/399c931706b682d22a4fa7f5f9326347c5cafc9f/abis/StablePool.json